tests/integration/test_valid_data_in_request.py

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8 -*-
+# SPDX-License-Identifier: MIT
+
+import utils
+import time
```

Comment:
The imports should be in sections of stdlib, third-party, and local imports.
```suggestion
import time

import utils
```